airflow/opt/providers/etna/etna/etls/metis_linker.py

```diff
@@ -16,6 +16,9 @@
 import numpy
 import secrets
 
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
There was no way previously to see stack traces from errors, which made debugging issues in prod impossible.